### PR TITLE
product card as well as img

### PR DESCRIPTION
**Pull request by Varsani2520**
i solve the card size issue as well as imagesnot displaying issue 

Fixes # (issue)
#369 

## Type of change

Please give a X on it which is applicable

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor Code
- [ ] A documentation update
- [ ] Others(mentioned in the issue number)

# Screenshots and Videos:

image

# Checklist:
Give a X on, which is applicable

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pooranjoyb — COMMENTED** [review]
Great job, @Varsani2520! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by Varsani2520** [comment]
i added img in supabase and remove dummy images as well as solve the size 

@pooranjoyb review it

File: src/pages/Shop/Shop.tsx [hunk 1]
```diff
@@ -17,7 +17,8 @@ function Shop() {
     console.log(data);
     if (data) {
       setProducts(data);
-      setSearchResults(data); // Initially display all products
+      setSearchResults(data); 
+      console.log("daata",data)// Initially display all products
```
Comment (pooranjoyb) [inline src/pages/Shop/Shop.tsx:21]: remove this console-log

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: src/pages/Shop/Shop.tsx (hunk 1 above)
Comment (Varsani2520) [reply src/pages/Shop/Shop.tsx:21]: Ok

**Comment by Varsani2520** [comment]
@pooranjoyb now check it

**Comment by pooranjoyb** [comment]
Also do #373 and #374 in this same PR only. @Varsani2520

**Comment by Varsani2520** [comment]
@pooranjoyb ,first this PR merged so i can work on #373 and #374

**Comment by pooranjoyb** [comment]
> @pooranjoyb ,first this PR merged so i can work on #373 and #374

yeh do those thanges in this same PR only.

File: src/pages/Profile/EditProfileModal.tsx [hunk 2]
```diff
@@ -19,15 +19,17 @@ interface Props {
     onUpdate: () => void;
 }
 
-const phoneSchema = z.string().refine((value) => validator.isMobilePhone(value), {
-    message: 'Invalid mobile phone number',
+const phoneSchema = z.string().refine((value) => {
```
Comment (pooranjoyb) [inline src/pages/Profile/EditProfileModal.tsx:21]: Hey @Varsani2520 we already have the schema for Phone number please check the Zod Validation for phone in utils

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: src/pages/Profile/EditProfileModal.tsx (hunk 2 above)
Comment (Varsani2520) [reply src/pages/Profile/EditProfileModal.tsx:21]: @pooranjoyb ,i added strict validation on phone schema user only allowed to 10 digit in mobile number

**Comment by Varsani2520** [comment]
@pooranjoyb ,i fixed #374 
image

File: src/pages/Profile/EditProfileModal.tsx (hunk 2 above)
Comment (pooranjoyb) [reply src/pages/Profile/EditProfileModal.tsx:21]: > @pooranjoyb ,i added strict validation on phone schema user only allowed to 10 digit in mobile number

Opps sorry, my bad :/ I thought u added manual validation.

**Pull request merged**